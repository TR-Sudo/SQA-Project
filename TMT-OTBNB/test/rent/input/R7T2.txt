login
user01
rent
a0b1c3d4
15
1
no
logout
quit