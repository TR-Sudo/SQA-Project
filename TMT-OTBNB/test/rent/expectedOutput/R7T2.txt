Enter transaction: Enter username: You have been logged in
Enter transaction: Enter Rental ID: Enter number of nights: Error: Enter valid range [1-14]
Enter number of nights: Confirm rental (Yes/No): Rental not booked
Enter transaction: You have been logged out
Enter transaction: 